tests/sles4sap/configure_selinux_policy.pm

```diff
@@ -0,0 +1,63 @@
+# Copyright 2018-2025 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Summary: Configure SELinux policy according to the release notes for SLES4SAP
+#16
+# Maintainer: QE SAP <qe-sap@suse.de>
+
+use base 'selinuxtest';
+use strict;
+use warnings;
+use testapi;
+use serial_terminal qw(select_serial_terminal);
+use version_utils qw(is_sle);
+use bootloader_setup qw(replace_grub_cmdline_settings);
+use power_action_utils;
+
+sub run {
+    my ($self) = @_;
+
+    return "SLES4SAP 15 does not support SELinux. Doing nothing." if is_sle('<16');
+    select_serial_terminal;
+
+    my $policy = get_var('SELINUX_POLICY') // 'permissive';
+    die "Invalid policy type." unless $policy =~ /permissive|enforcing|disabled/;
+    record_info("Policy:$policy");
+
+    # selinuxtest does not provide an option to disable SELinux, yet, so we have
+    # to implement that option here.
+    if ($policy eq 'disabled') {
+        replace_grub_cmdline_settings('security=selinux', 'security=apparmor', update_grub => 0);
+        replace_grub_cmdline_settings('selinux=1', 'selinux=0', update_grub => 1);
+        power_action('reboot');
+        opensusebasetest::wait_boot(opensusebasetest->new(), bootloader_time => 200);
+        select_serial_terminal;
```

Comment:
Wondering if a call to `reset_consoles` is required here.

Reply:
I copy pasted the code from update_install so on the one hand, I am cargo culting but on the other hand the cult reboots very succesfuly.